__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("STATUS 200: returns an array of topic objects with correct properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const topics = response.body.topics;
+
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
```

Comment:
You could also use the .toMatchObject jest matcher here. 
For example, 
`expect(topic).toMatchObject({
            slug: expect.any(String),
            description: expect.any(String),
          });`

Reply:
done! thank you :)